modules/message.sh

```diff
@@ -59,7 +59,37 @@ function message {
           done
           ;;
         *)
-          echo "This command does not exist, please look at the following:"
+          echo "this command does not exist, please look at the following:"
+          message_help
+          ;;
+      esac
+      ;;
+    slack)
+      case "$2" in
+        apikey)
+          echo "Please look at the following to see how to get the key:"
+          message_help
+          echo "Enter your key: "
+          read -r stoken
```

Comment:
pretty sure a non-interactive approach will do better if we're integrate this with treehouses remote